MIGRATION.md

```diff
@@ -620,6 +579,47 @@ The SDK no longer filters out health check transactions by default. Instead, the
 by the Sentry backend by default. You can disable dropping them in your Sentry project settings. If you still want to
 drop specific transactions within the SDK you can either use the `ignoreTransactions` SDK option.
 
+## Next.js SDK Changes
+
+### Removal of the `sentry` property in your Next.js options
```

Comment:
Was this deprecated in v7?

We should put this under this section: https://github.com/getsentry/sentry-javascript/blob/8e0d24369f96145ad78a4983aab31583fbf8f358/MIGRATION.md#nextjs-sdk

Reply:
Not strictly deprecated but we haven't been recommending this way of setting up the SDK for longer than a year.

I moved the section! Thanks!